docs/getting-started/tutorial-sample-filter.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[tutorial-sample-filter]]
+=== Filtering the data
+
+Many visualizations in the *Global Flight* dashboard are interactive. They offer 
+mouse clicks and other controls for you to filter the view of the data. Filters 
+are global&mdash;a change in one visualization simultaneously changes the data 
```

Comment:
The "offer mouse clicks" phrasing is awkward. Maybe, just "You can apply filters to modify the view of the data across all visualizations."?